packages/password-field/test/password-field.test.js

```diff
@@ -2,9 +2,7 @@ import { expect } from '@esm-bundle/chai';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import { fixtureSync, focusout } from '@vaadin/testing-helpers';
-import { PasswordField } from '../src/vaadin-password-field.js';
-
-customElements.define('vaadin-password-field', PasswordField);
+import '../src/vaadin-password-field.js';
```

Comment:
```suggestion
import '../vaadin-password-field.js';
```

Reply:
We should test the `src` version of components to avoid relying on Lumo (some tests have this problem).
So I have updated the new implementations to use the unstyled version, for example:

https://github.com/vaadin/web-components/blob/a7d2733c55c65237288d10ed47f4c25919430761/packages/text-area/test/text-area.test.js#L4